vignettes/prompt-design.Rmd

```diff
@@ -251,10 +251,10 @@ chat <- chat_openai(c(instruct_json, instruct_weight))
 chat$chat(ingredients)
 ```
 
-Just providing the examples seems to work remarkably well. But I found it useful to also include description of what the examples are trying to accomplish. I'm not sure if this helps the LLM or not, but it certainly makes it easier for me to understand the organisation of the whole prompt and check that I've covered the key pieces that I'm interested in.
+Just providing the examples seems to work remarkably well. But I found it useful to also include a description of what the examples are trying to accomplish. I'm not sure if this helps the LLM or not, but it certainly makes it easier for me to understand the organisation of the whole prompt and check that I've covered the key pieces I'm interested in.
 
 ```{r}
-#| cacahe: false
+#| cache: false
```

Comment:
Ooops

Reply:
That reminds me. Up on Line 40 setup chunk, I added an "echo: false" to hide what seemed to me to be a single stray line of code: 

"library(elmer)"

Should that actually be visible (without  "echo: false")? I think I saw the same some other documents/vignettes.

Reply:
Yeah, I think it should be visible. In general, we like to make it obvious that we're loading the package at the start of vignettes.